client/scss/components/_user-menu.scss

```diff
@@ -74,20 +74,17 @@ $user-menu-border-width-mobile: 5px;
     }
 
     @media not screen and (max-width: $mobile-max-width) {
-      right: calc(#{$avatar-width} / 2 + #{$avatar-margin});
```

Comment:
Why was this change necessary? If the margin on .user-menu changes, menu will no longer adjust.

Reply:
😕 

WHHOOOPPPSSSS. That wasn't meant to stay :/

I set it to exact values so I could mess with the position of the menu, but never moved it back :/